src/screens/AuthLanding.js

```diff
@@ -1,29 +1,28 @@
 import React from 'react';
-import {
-  StyleSheet, Text, View, TouchableOpacity,
-} from 'react-native';
-import Colors from '../constants/Colors';
-
+import { StyleSheet, View } from 'react-native';
+import AppButton from '../components/AppButton';
+import Spacer from '../components/Spacer';
 
 const AuthLandingScreen = ({ navigation }) => (
   <View style={styles.container}>
-    <TouchableOpacity
-      style={[styles.button, styles.signUpButton]}
-      onPress={() => {
+    <AppButton
+      buttonStyle="primaryButton"
+      textStyle="primaryText"
+      title="Sign Up"
+      onSubmit={() => {
         navigation.navigate('SignUp');
       }}
-    >
-      <Text style={[styles.buttonText, styles.signUpButtonText]}>Sign up</Text>
-    </TouchableOpacity>
-    <TouchableOpacity
-      style={styles.button}
-      onPress={() => {
+    />
+    <Spacer />
```

Comment:
would be great if the spacer could take in a "height" attribute.

Reply:
I'll change that so it there's a default set height but you can pass a height `prop` in to change that default.